src/odemis/gui/test/comp_canvas_cairo_test.py

```diff
@@ -406,7 +406,7 @@ def test_nanana(self):
 
         # old_canvas = DraggableCanvas(self.panel)
         tab = self.create_simple_tab_model()
-        mpp = FloatContinuous(10e-6, range=(1e-3, 1), unit="m/px")
+        mpp = FloatContinuous(10e-6, range=(1e-6, 1), unit="m/px")
```

Comment:
Not related to the change but maybe we should at a comment on why this test is called nanana at the start of this test.
Even after looking at the code it isn't 100% clear to me (yet)